src/apps/wallet/ecdh.py

```diff
@@ -28,8 +29,10 @@
 
 
 async def require_confirm_ecdh_session_key(ctx, identity):
-    lines = chunks(identity, 18)
-    content = Text('Decrypt', ui.ICON_DEFAULT, ui.MONO, *lines, max_lines=5)
+    lines = chunks(serialize_identity_without_proto(identity), 18)
+    header = 'Decrypt {}'.format(identity.proto.upper()
```

Comment:
Style nitpick: please use modulo formatting for consistency.

Reply:
Done.